system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -85,6 +87,12 @@ LinuxCpuMeasurementNode::LinuxCpuMeasurementNode(
   last_measurement_()
 {}
 
+bool LinuxCpuMeasurementNode::setupStart()
+{
+  last_measurement_ = ProcCpuData();
```

Comment:
Why not initialize in the constructor?

Reply:
This is for the restart case. If you do `start()` -> `stop()` -> `start()`, you shouldn't have the first `MetricsMessage` you send after the second time you call `start()` use (stale) measurements from before the first time you called `stop()`.